src/component/MovieCardListRefetch.js

```diff
@@ -0,0 +1,23 @@
+import MovieCard from "./MovieCard";
+
+/**
+ * cardDatas : [
+ *  {
+ *    href : 영화 상세페이지 url
+ *    src : 영화 썸네일 url
+ *    title : 영화 이름
+ *    alt : 영화 썸네일 alt
+ *    score : 영화 점수
+ *  }
+ * ]
+ */
+export default ({ cardDatas }) => {
+  const ul = document.querySelector("ul");
+
+  cardDatas.map((cardData) => {
+    const { href, src, title, alt, score } = cardData;
+    const card = MovieCard({ href, src, title, alt, score });
```

Comment:
굳이 해체해서 넣어줄 필요가 있을까요?
```suggestion
    const card = MovieCard(cardData);
```
이러면 더 좋지 않을까 싶네요!

Reply:
수정했습니다.